scripts/oidc-provider.js

```diff
@@ -1,10 +1,26 @@
-import { Provider } from 'oidc-provider';
+import { Provider, interactionPolicy } from 'oidc-provider';
+
+const { base, Prompt, Check } = interactionPolicy;
+
+const policy = base();
+policy.add(
+  new Prompt(
+    { name: 'noop', requestable: false },
+    new Check('foo', 'bar', ctx => {
+      if (ctx.query?.scope?.includes('offline_access')) {
+        ctx.oidc.params.scope = `${ctx.oidc.params.scope} offline_access`;
+      }
+      return Check.NO_NEED_TO_PROMPT;
+    })
+  ),
+  0
+);
```

Comment:
By default, Node-OIDC does not issue a refresh token unless you set `prompt=consent`. This is a bit different as to how Auth0 works and was causing some troubles with some tests.

This change now ensures refresh tokens are issues whenever `offline_access` is requested as a scope.